frontend/lib/components/utils/task_tree.dart

```diff
@@ -16,53 +18,61 @@ class TaskTree extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      children: [
-        if (task.author != null)
-          Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              const Text(
-                'Créée par ',
-                style: TextStyle(
-                  fontWeight: FontWeight.w500,
-                  fontSize: 14,
-                ),
-              ),
-              Text(
-                '${task.author}',
-                style: const TextStyle(
-                  fontSize: 14,
-                  fontWeight: FontWeight.w700,
-                ),
+    return GetBuilder<EditTaskController>(
+      init: EditTaskController(task: task),
+      builder: (editTask) {
+        Get.lazyPut(() => EditTaskController(task: task));
+        return Column(
+          children: [
+            const SizedBox(height: 16),
+            if (editTask.task.author != null)
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  const Text(
+                    'Créée par ',
```

Comment:
It's not in english?

Reply:
No, as seen before with @PtitLuca the frontend has been developed in french. It has not been expected to be developed in English, but in the future, thanks to GetX, it would be relatively easy to enable an English option.